wagtail/blocks/field_block.py

```diff
@@ -24,6 +23,12 @@
 
 from .base import Block
 
+try:
+    from django.forms.fields import CallableChoiceIterator
+except ImportError:
+    # DJANGO_VERSION >= 5.0
+    from django.utils.choices import CallableChoiceIterator
```

Comment:
nitpick: we normally try the new version first, then default to the older ones.
Is there any reason not to do se here as well?

Other than that LGTM

Reply:
Oops, yeah just double-checked. Thanks, fixed it